### PR TITLE
Adds Cookie Properties for Distributed Session

### PR DESCRIPTION
**Pull request by tsschmidt**
The distributed session stores needs to set a common path for the DISESSION cookie since OIDC requests fall back to the default /oauth2.0 callback for authorization.  Setting cookies set to context request path causes to /oidc request to fail since the set cookie is not forwarded to to callback endpoint which is under /oauth2.0.

### CODE REVIEW

File: api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/CasConfigurationProperties.java [hunk 1]
```diff
@@ -242,6 +243,11 @@
     @NestedConfigurationProperty
     private WarningCookieProperties warningCookie = new WarningCookieProperties();
 
+    /**
+     * Distributed OAuth Session cookie.
+     */
```
Comment (mmoayyed) [inline api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/CasConfigurationProperties.java]: `@NestedConfigurationProperty`

File: api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/CasConfigurationProperties.java [hunk 2]
```diff
@@ -242,6 +243,11 @@
     @NestedConfigurationProperty
     private WarningCookieProperties warningCookie = new WarningCookieProperties();
 
+    /**
+     * Distributed OAuth Session cookie.
```
Comment (mmoayyed) [inline api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/CasConfigurationProperties.java]: This is not entirely accurate, as the distributed-session store is not exclusively used for OAuth/OIDC but also delegated authn.

**Comment by mmoayyed** [comment]
@leleuj Wanted to make sure you are aware of the change here as well.

**Comment by leleuj** [comment]
Got it

**Pull request merged**